app/views/notifiers/course/video_notifier/attempted/course_notifications/_feed.html.slim

```diff
@@ -4,8 +4,7 @@
 = div_for(notification) do
   // TODO: Display user image here.
   - user_link = link_to_user(activity.actor)
-  // TODO: Enable video_link when routes are fixed
-  // - video_link = link_to video.title, course_video_path(notification.course, video)
-  - video_link = video.title
+  - video_link = link_to(video.title,
+                         new_course_video_submission_path(notification.course, video))
```

Comment:
Suggest that we direct to the `show` action instead of going to the editing of the submission. Some admins (who are not course_users) can see this feed.

Reply:
That'll work.

However, do you think it'll be better if I display different links for different people? A course user lands on a submission page while an admin lands on `show`. I'm thinking of doing it via a `can?(:attempt)` or something similar.

The reason being is that most of the video activities are in a submission edit, and it'll be better if links land there as much as possible. (Prof wants the lesson plan link to land on the video player page directly for example.)

Reply:
Would prefer show` page, although it's an extra click to get into the submission page. 

If we do direct them into the `submission edit`, we might end up creating submissions unknowingly for certain students (who do not have that intention). Most of the time if I see this activity in the feed and click it, it's because I probably haven't seen this assessment / video before, so good to take a look at the due dates, rather than trying to attempt it (that will usually happen in the videos index page. 

For the links on lesson plan, I would suggest that you have links to the video's `show` page and `edit` submission page, and make that link/button explicit.

Reply:
Alright, that sounds good, will fix.